### PR TITLE
Fix/dialogs behind keyboard on Android and IOS

### LINKED ISSUE #1093: dialogs should shrink and positioned on the top of the screen when keyboard is open (android)
On Android Mobile app. Sometimes, depending on the screen dimensions. The Login and Signup dialog is behind the keyboard. 
imagen

### PR DESCRIPTION
**Pull request by tomivm**
This PR uses a newly added plugin to adjust welcome dialog sizes to not be covered by the keyboard on Android and IOS. Please @RodriSanchez1 can you Test it? Also, please use the keyboard with live mode to make sure the board is not resized.

### CODE REVIEW

File: src/components/WelcomeScreen/WelcomeScreen.container.js [hunk 1]
```diff
@@ -35,6 +45,18 @@ export class WelcomeScreen extends Component {
     onClose: PropTypes.func
   };
 
+  handleKeyboardDidShow = event => {
+    this.setState({
+      keyboard: { isKeyboardOpen: true, keyboardHeight: event.keyboardHeight }
+    });
+  };
+
+  handlekeyboardDidHide = () => {
```
Comment (RodriSanchez1) [inline src/components/WelcomeScreen/WelcomeScreen.container.js]: change `handlekeyboardDidHide`  to `handleKeyboardDidHide`  ( Camel case)

File: src/components/WelcomeScreen/WelcomeScreen.container.js [hunk 2]
```diff
@@ -115,9 +137,60 @@ export class WelcomeScreen extends Component {
...
+      removeEvent: true
+    });
+  }
+
   render() {
     const { finishFirstVisit, heading, text, onClose } = this.props;
-    const { activeView } = this.state;
+    const { activeView, dialogWithKeyboardStyle } = this.state;
+
+    //const dialogContentWithKeyboardStyle = this.dialogContentStyle();
```
Comment (RodriSanchez1) [inline src/components/WelcomeScreen/WelcomeScreen.container.js]: Remove unncesary comment

**Review by RodriSanchez1 — APPROVED** [review]
Its a good feature! I like it a lot!

**Comment by RodriSanchez1** [comment]
Suggestion: 
- Add an scroll indicator on sign up dialog.  Needs to scroll to see the action buttons! 
Image on Samsung Galaxy Tab A7
Screenshot_20230802-205517_AAC_Cboard 1

- Smartphones will have problems on rotated screen. But i think this is not a big problem, we can avoid it. See:

Screenshot_20230802_205032_AAC Cboard

**Review by martinbedouret — CHANGES_REQUESTED** [review]
@tomivm Please review comments from rodri

**Review by martinbedouret — APPROVED** [review]

**Pull request merged**